src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -356,6 +356,11 @@ configurationRegistry.registerConfiguration({
 			'minimum': 10,
 			'description': localize('notebook.maxRichTextUndoHistory', "The maximum number of changes stored in the undo history for the notebook Rich Text editor.")
 		},
+		'notebook.renderTablesInHtml': {
+			'type': 'boolean',
+			'default': false,
+			'description': localize('notebook.renderTablesInHtml', "Tables are displayed as HTML in text cells.")
```

Comment:
Not sure about the exact wording but we should call out html vs markdown table. Maybe something like: `Keep HTML tables in text cells instead of converting to markdown tables.`